pkg/containers/client.go

```diff
@@ -116,6 +118,30 @@ func (c *Client) LoadContainerTasks(ctx context.Context) ([]ContainerProcess, er
 	return result, nil
 }
 
+func (c *Client) LoadContainers(ctx context.Context) error {
+	containersList, err := c.criRuntimeServiceClient.ListContainers(ctx, &criapi.ListContainersRequest{
```

Comment:
Do we need to invalidate the containers we already have in the map here?

Reply:
Yep, cleaning up cached containers would cleanup some old dangling containers in case we miss rmdir event